background/main.js

```diff
@@ -1,11 +1,22 @@
 var SERVER_URL = "https://chronicle.dev.mozaws.net";
```

Comment:
Last time I checked, you had to roll your own settings page for user configurable prefs, which is why it wasn't part of this initial effort. Alternatively, you can compile different extensions with different values for this.